direct/nn/vsharp/vsharp.py

```diff
@@ -51,6 +50,8 @@ def __init__(
             Tuple of integers specifying the dilation factor for each convolutional layer in the network.
         multiscale_depth : int
             Number of multiscale features to include in the output. Default: 1.
+        activation : ActivationType
+            Activation function to use on the output. Default: ActivationType.PRELU.
```

Comment:
Does this parse properly in the documentation, or is it possible to have link to the `ActivationType`?

Reply:
Yes it parses fine!